api/src/pcapi/serialization/decorator.py

```diff
@@ -135,7 +135,11 @@ def sync_validate(*args: Any, **kwargs: Any) -> Response:
                 logger.info(
                     "Error when decoding request body: %s",
                     error,
-                    extra={"contentTypeHeader": request.headers.get("Content-Type"), "path": request.path},
+                    extra={
+                        "contentTypeHeader": request.headers.get("Content-Type"),
+                        "path": request.path,
+                        "data": request.data[:1000],
```

Comment:
https://flask.palletsprojects.com/en/2.2.x/api/#flask.Request.get_data 
It is advised to limit request data since a client could send "dozens of megabytes"